src/content/developers/docs/scaling/state-channels/index.md

```diff
@@ -6,71 +6,256 @@ sidebar: true
 incomplete: true
 sidebarDepth: 3
 ---
-
-State channels allow participants to transact `x` number of times off-chain while only submitting two on-chain transactions to the Ethereum network. This allows for extremely high transaction throughput.
+State channels allow participants to securely transact off-chain while keeping interaction with Ethereum Mainnet at a minimum. Channel peers can conduct an arbitrary number of off-chain transactions while only submitting two on-chain transactions to open and close the channel. This allows for extremely high transaction throughput and results in lower costs for users. 
 
 ## Prerequisites {#prerequisites}
+You should have read and understood our pages on [Ethereum scaling](/developers/docs/scaling/) and [layer 2](/layer-2/).
+
+## What are channels? {#what-are-channels}
+
+Public blockchains, such as Ethereum, experience scalability issues due to their distributed architecture: on-chain transactions must be executed by all nodes, even if the said transactions concern a small number of users. Blockchain channels solve this problem by allowing users to keep interactions off the main blockchain.
+
+Channels are simple bi-directional, peer-to-peer protocols that allow two parties to execute transactions off-chain. A channel functions as a fraud proof-based, multisignature state machine replicated by two P2P nodes. 
```

Comment:
This is extremely jargon-heavy and probably doesn't help someone trying to learn about state-channels for the first time.
```suggestion
Channels are simple bi-directional, peer-to-peer protocols that allow two parties to make many transactions between themselves and then only post the final results to the blockchain. The channel uses cryptography to demonstrate that the summary data they generate is truly the result of a valid set of intermediate transactions. A ["multisig"](developers/docs/smart-contracts/#multisig) smart contract ensures the transactions are signed by the correct parties.
```